packages/shared-data/extensions.json

```diff
@@ -396,7 +396,7 @@
     "link": "/guides/database/extensions/uuid-ossp"
   },
   {
-    "name": "pgvector",
+    "name": "vector",
```

Comment:
this fixes the issue that the vector extension card link wasn't being displayed.